hashes/zkevm-keccak/src/keccak_packed_multi.rs

```diff
@@ -1698,6 +1719,20 @@ pub fn keccak_phase0<F: Field>(
             // Absorb data
             let absorb_from = cell_manager.query_cell_value();
             let absorb_data = cell_manager.query_cell_value();
+            // The cell above holds a word which will be absorbed. Therefore
+            // we add it to `flagged_words`.
+            if round < NUM_WORDS_TO_ABSORB {
+                let cell_offset = absorb_data.rotation as usize;
+                let cell_column_idx = absorb_data.column_idx;
+                // We compute the row index from the cell offset, the current
+                // round and the current chunk
+                let row_index = idx * (NUM_ROUNDS + 1) * num_rows_per_round
+                    + round * num_rows_per_round
+                    + cell_offset;
```

Comment:
This computation happens in at least one other place so let's extract it into its own function. 

(I know, I know, it seems futile to start doing this kind of thing when you've inherited a 400+ line function to begin with...)